documentation/src/content/docs/en/essentials/manage-states.mdx

```diff
@@ -0,0 +1,240 @@
+---
+title: State Management in effector
+description: How state works in effector and how to manage it
+---
+
+import Tabs from "../../../../components/Tabs/Tabs.astro";
+import TabItem from "../../../../components/Tabs/TabItem.astro";
```

Comment:
What do you think add an alias for this?

https://docs.astro.build/en/guides/imports/#aliases

```suggestion
import { Tabs, TabItem } from "@components/Tabs";
```

Reply:
good thought